src/components/ArticlesList.jsx

```diff
@@ -1,31 +1,34 @@
 import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
 import { getArticles } from "../../api";
 
 export default function ArticlesList() {
-  const [articles, setArticles] = useState([]);
+  const [articlesList, setArticlesList] = useState([]);
   const [loaded, setLoaded] = useState(false);
 
   useEffect(() => {
     getArticles().then((results) => {
       setLoaded(true);
-      setArticles(results);
+      setArticlesList(results);
     });
   }, []);
 
   if (loaded) {
     {
       return (
         <section className="articles">
-          {articles.map((article) => {
+          {articlesList.map((article) => {
             return (
               <section key={article.article_id} className="article">
-                <p>{article.title}</p>
+                <Link to={`/articles/${article.article_id}`}>
+                  {article.title}
+                </Link>
                 <img src={article.article_img_url} style={{ width: "70%" }} />
-                <p>{article.author}</p>
-                <p>{article.topic}</p>
-                <p>{article.votes}</p>
-                <p>{article.comment_count}</p>
-                <p>{article.created_at}</p>
+                <p>Written by: {article.author}</p>
+                <p>Topic: {article.topic}</p>
+                <p>Votes: {article.votes}</p>
+                <p>Comments: {article.comment_count}</p>
+                <p>Created: {article.created_at}</p>
```

Comment:
It would be better to have the timestamp formatted in a bit more of a human-readable way, than the default. Can you think of a way to do this? 💡